.github/ISSUE_TEMPLATE/config.yml

```diff
@@ -0,0 +1,5 @@
+blank_issues_enabled: false
+contact_links:
+  - name: Support Request
+    url: https://discord.gg/hyperledger
```

Comment:
This needs to be updated to the LFDT discord URL

Reply:
Actually just validated that is work! So just the email below that needs changing maybe

Reply:
we can't change that link for boring reasons